assets/components/atoms/event-teaser/event-teaser-highlighted.twig

```diff
@@ -0,0 +1,24 @@
+<a href="#" class="event-teaser event-teaser-highlighted card p-4">
+
+  <picture class="card-img-top">
+    <img src="./images/event_teaser_large.jpg" class="d-block" title="professor Xile Hu" aria-labelledby="event-teaser-label" />
+  </picture>
+  <p id="event-teaser-label" class="sr-only">a portrait of professor Xile Hu</p>
+
+  <div class="card-body mt-2">
+    <h3 class="card-title">Data Science and Mobility Conference</h3>
+    <p>The goal of the Data Science and Mobility Conference is to look at specific examples to find out what role data science has to play in tackling the challenges facing the mobility of the future.</p>
+    <div class="row text-small">
+      <div class="col-md-6">
+        <p class="mb-0">Conférences - Séminaires</p>
```

Comment:
To many redondant classes. You can do something nicer like : 

```scss
.event-teaser .card-body p { margin-bottom: 0; }
```

Reply:
Indeed, following our discussions, it will be better to have a maximum of clean markup to make usage of the styleguide easier for other developpers. Will be changed